src/main/resources/templates/db_scripts/schema.json

```diff
@@ -80,6 +80,11 @@
       "run": "after",
       "snippetPath": "budget_encumbrances_rollover.sql",
       "fromModuleVersion": "mod-finance-storage-7.1.1"
+    },
+    {
+      "run": "after",
+      "snippetPath": "migration/restrictions_for_fund_and_ec_codes.sql",
+      "fromModuleVersion": "mod-finance-storage-7.2.0"
```

Comment:
Please change it to 8.0.0. As major version of the API was changed then major version of the module should be released